SU-SWS/stanford_sites_helper
# SITES-1223: Add check for Acquia hosting and display help block there.

# READY FOR REVIEW

# Summary
- Display help block on Acquia hosting

# Needed By (Date)
- This week would be good

# Criticality
- How critical is this PR on a 1-10 scale? 4/10
- Our self-service sites have been missing the link to our SNOW form since July.

# Steps to Test

1. Set an environment variable `AH_SITE_ENVIRONMENT`
2. `drush -y si stanford`
3. Check that you see the " Problems using this service? Submit a help ticket." block

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket: SITES-1223

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
